lib/naver/searchad/api/core/base_service.rb

```diff
@@ -1,3 +1,11 @@
+require 'addressable/uri'
+require 'addressable/template'
+require 'httpclient'
+require 'naver/searchad/api/options'
```

Comment:
Using require_relative allows the code to operate even if not installed as a gem. Which is quite handy when you develop on it if you ask me :)